lighthouse-core/config/config.js

```diff
@@ -381,6 +383,30 @@ class Config {
     return mergedItems;
   }
 
+  /**
+   * Observed throttling methods (devtools/provided) require at least 5s of quiet for the metrics to
+   * be computed. This method adjusts the quiet thresholds to the required minimums if necessary.
+   *
+   * @param {LH.Config.Json} config
+   */
+  static adjustDefaultPassForThrottling(config) {
+    if (config.settings.throttlingMethod !== 'devtools' &&
+        config.settings.throttlingMethod !== 'provided') {
+      return;
+    }
+
+    const defaultPass = config.passes.find(pass => pass.passName === 'defaultPass');
+    if (!defaultPass) return;
+
+    const overrides = constants.observedPassConfigOverrides;
+    defaultPass.pauseAfterLoadMs =
```

Comment:
i wouldn't yell bloody murder if you renamed this to `dPass` or something to avoid all the wrapppppppppppppppping

Reply:
heh, yeah I spent more time than I should to save this one, basically nothing saves the last one (triple `networkQuietThresholdMs` is just too much` so I gave up, at least here it's consistent and easy to understand